File: src/MyChess/Interfaces/UserSettings.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace MyChess.Interfaces
{
    public class UserSettings
    {
        [JsonPropertyName("id")]
        public string ID { get; set; } = string.Empty;

        [JsonPropertyName("playAlwaysUp")]
        public bool PlayAlwaysUp { get; set; }

        [JsonPropertyName("notifications")]
        public List<UserNotifications> Notifications { get; set; } = new List<UserNotifications>();
    }
}

File: src/MyChessReact/src/models/UserSettings.ts
import { UserNotifications } from "./UserNotifications";

export class UserSettings {
    id: string;

    playAlwaysUp: boolean;
    notifications: UserNotifications[];

    constructor() {
        this.id = "";

        this.playAlwaysUp = false;
        this.notifications = [];
    }
}
